.github/workflows/test-stellar.yaml

```diff
@@ -0,0 +1,236 @@
+              --protocol-version 22
+
+          RPC_URL="http://localhost:8000/soroban/rpc"
+          NETWORK_PASSPHRASE="Standalone Network ; February 2017"
+          echo "RPC_URL=${RPC_URL}" >> $GITHUB_ENV
+          echo "NETWORK_PASSPHRASE=${NETWORK_PASSPHRASE}" >> $GITHUB_ENV
+
+      - name: Add as configured network
+        shell: bash
+        run: |
+          stellar network add local \
+              --rpc-url "$RPC_URL" \
+              --network-passphrase "$NETWORK_PASSPHRASE"
+
+      - name: Prepare local.json
+        run: |
+          echo '{
+              "chains": {
+                  "stellar": {
+                      "name": "Stellar",
+                      "axelarId": "stellar",
+                      "networkType": "local",
+                      "chainType": "stellar",
+                      "tokenSymbol": "XLM",
+                      "rpc": "http://127.0.0.1:8000/soroban/rpc",
+                      "horizonRpc": "http://127.0.0.1:8000",
+                      "contracts": {}
+                  }
+              }
+          }' > ./axelar-chains-config/info/local.json
+
+      - name: Display local.json
+        run: cat ./axelar-chains-config/info/local.json
+
+      ###### Acquire WASM Contracts   ######
+      - name: Configure CF credentials
+        run: |
+          cd $HOME; mkdir ~/.aws; touch ~/.aws/credentials; touch ~/.aws/config
+          echo "[default]
+          aws_access_key_id = ${{ secrets.cf-bucket-access-key-id }}
+          aws_secret_access_key = ${{ secrets.cf-bucket-secret-access-key }}" > ~/.aws/credentials
+          echo "[default]
+          region=auto
+          output=json" > ~/.aws/config
+
+      - name: Fetch WASM contracts from CF
+        env:
+          S3_BUCKET_NAME: ${{ vars.CF_BUCKET_NAME }}
+          ENDPOINT_URL: ${{ secrets.cf-endpoint-url }}
+          CF_BUCKET_ROOT_KEY: ${{ vars.CF_BUCKET_ROOT_KEY }}
+        run: |
+          export CF_WASM_BUCKET_ROOT_KEY="${CF_BUCKET_ROOT_KEY}/${{ env.PACKAGE_NAME }}/${{ env.PACKAGE_VERSION }}/wasm"
+          for name in axelar_gateway axelar_operators axelar_gas_service; do
+            aws s3api get-object --bucket $S3_BUCKET_NAME --key "$CF_WASM_BUCKET_ROOT_KEY/$FILE_KEY" --body "$name.optimized.wasm" --acl public-read --endpoint-url $ENDPOINT_URL
+          done
+
+          ls
+
+          WASM_DIR=./
+          echo "WASM_DIR=${WASM_DIR}" >> $GITHUB_ENV
+
+      - name: Setup Stellar wallet
+        run: |
+          stellar keys generate test \
+              --rpc-url "$RPC_URL" \
+              --network-passphrase "$NETWORK_PASSPHRASE"
+
+          STELLAR_PRIVATE_KEY=$(stellar keys show test)
+          STELLAR_ADDRESS=$(stellar keys address test)
+          echo "STELLAR_PRIVATE_KEY=${STELLAR_PRIVATE_KEY}" >> $GITHUB_ENV
+          echo "STELLAR_ADDRESS=${STELLAR_ADDRESS}" >> $GITHUB_ENV
+
+      - name: Fund Stellar wallet
+        run: node stellar/faucet --recipient $STELLAR_ADDRESS
```

Comment:
Why don't we just use `--fund` parameter to fund the stellar wallet?
You can test it with
```
stellar keys generate --global test10 --network testnet --fund
stellar keys address test10
```
and go to stellar.expert and check the balance of the wallet address.

Reply:
it is possible to fund it during wallet creation but I was thinking that this would act like a test for the `faucet` command.